src/frontend/js/actions/users-actions.js

```diff
@@ -0,0 +1,18 @@
+import {
```

Comment:
If you don't have to look at the api.js file, this part looks really nice. Even better than with promise or thunk middlewares